src/command_modules/azure-cli-iot/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.1.11 (2017-08-18)
```

Comment:
This should say 'unreleased'. Also, in setup.py, the version will be updated automatically when we have the next release.